spyder/app/mainwindow.py

```diff
@@ -872,11 +872,10 @@ def create_edit_action(text, tr_text, icon):
             self.outlineexplorer = OutlineExplorer(self)
             self.outlineexplorer.register_plugin()
 
-        if is_anaconda():
-            from spyder.widgets.status import CondaStatus
-            self.conda_status = CondaStatus(self, status,
-                                            icon=ima.icon('environment'))
-            self.conda_status.update_interpreter(self.get_main_interpreter())
+        from spyder.widgets.status import InterpreterStatus
+        self.conda_status = InterpreterStatus(self, status,
+                                              icon=ima.icon('environment'))
+        self.conda_status.update_interpreter(self.get_main_interpreter())

```

Comment:
We should also change the name of this variable to 

`self.interpreter_status`

Reply:
Will do.